app/src/main/java/com/nowak/wjw/simplecompass/ui/main/MainViewModel.java

```diff
@@ -1,32 +1,34 @@
 package com.nowak.wjw.simplecompass.ui.main;
 
-import android.hardware.Sensor;
-import android.hardware.SensorEvent;
-import android.hardware.SensorManager;
-import android.location.Location;
-
 import androidx.lifecycle.LiveData;
 import androidx.lifecycle.MutableLiveData;
 import androidx.lifecycle.Transformations;
 import androidx.lifecycle.ViewModel;
 
-import com.nowak.wjw.simplecompass.CompassStateEnum;
-import com.nowak.wjw.simplecompass.Event;
 import com.nowak.wjw.simplecompass.R;
+import com.nowak.wjw.simplecompass.data.LocationCoordinates;
+import com.nowak.wjw.simplecompass.domain.GetAzimuthUseCase;
+import com.nowak.wjw.simplecompass.domain.GetDestinationBearingUseCase;
+import com.nowak.wjw.simplecompass.domain.InitiateLastLocationUseCase;
+import com.nowak.wjw.simplecompass.domain.RequestAndStopLocationUpdatesUseCase;
+import com.nowak.wjw.simplecompass.domain.StartStopSensorListenerUseCase;
 
 import timber.log.Timber;
 
```

Comment:
Reviewers comment - "You use ViewModel where most of the logic dwells, but it’s strongly coupled to Android Data Binding, so I can’t take it as an MVVM."

I don't understand that